classes/01class/exercises/c01-git01/mail2kvanitha/README.md

```diff
@@ -0,0 +1,33 @@
+# c01-git01
```

Comment:
This file should not be part of the c01-git02 exercise delivery

Reply:
projects/project02/application/web/public/favicon.ico file removed from projects